oracle/oracle.go

```diff
@@ -41,8 +41,8 @@ import (
 )
 
 // Option represents available options for the oracle.Oracle.
+// TODO: remove this struct
 type Option struct {
-	TxnScope string
 }
```

Comment:
Why not just remove it in this PR?

Reply:
To minimize the change.. It has hundreds of usages. I prefer keeping this PR (relatively) small